.github/workflows/naughty-prune.yml

```diff
@@ -1,12 +1,16 @@
 name: prune naughties
 on:
   schedule:
-    - cron: '30 1 * * *'
+    - cron: '30 1 * * 0'
   # can be run manually on https://github.com/cockpit-project/bots/actions
   workflow_dispatch:
 jobs:
   maintenance:
     runs-on: ubuntu-latest
+    permissions:
+      issues: read
+      pull-requests: write
```

Comment:
I guess this is going to end up pushing a branch and opening a PR, but not triggering the test runs (which are required for merging)...

Reply:
Ah right, only the LGTM ones. So I suppose we'll need to create a deploy key and a "self" environment for bots.

Reply:
Reworked, I updated the description for the new test run and resulting PR.